README.md

```diff
@@ -211,22 +211,14 @@ pre-built libraries are only needed for unit tests, Python, gRPC, and Comm
 support. If Boost or Python libraries are not found on the system, then some
 tests and examples will not be built.
 
-To build Bond's gRPC++ integration from source, some of
-[gRPC's prerequisites](https://github.com/grpc/grpc/blob/master/INSTALL.md#building-using-cmake-with-boringssl)
-are also needed:
-
-```bash
-choco install activeperl golang ninja yasm
-```
-
-In order to generate a solution to build the C++ and Python versions with Visual
+In order to generate a solution to build the Bond Core C++ and Python with Visual
 Studio 2015 run the following commands from the root `bond` directory:
 
 ```bash
 mkdir build
 cd build
 set PreferredToolArchitecture=x64
-cmake -G "Visual Studio 14 2015 Win64" ..
+cmake -DBOND_ENABLE_COMM=FALSE -DBOND_ENABLE_GRPC=FALSE -G "Visual Studio 14 2015 Win64" ..
```

Comment:
Consider swapping these `-D`s in both uses in this file - GRPC is the more interesting one.